src/libraries/System.Private.CoreLib/src/System/Globalization/DateTimeFormat.cs

```diff
@@ -1112,6 +1112,233 @@ private static StringBuilder FormatStringBuilder(DateTime dateTime, ReadOnlySpan
             return FormatCustomized(dateTime, format, dtfi, offset, result: null);
         }
 
+        internal static bool IsValidCustomDateFormat(ReadOnlySpan<char> format, bool allowThrow)
```

Comment:
Nit: I see we name the bool to drive the throws in other places, `shouldThrow` or `throwOnError`.

Reply:
will do